coverband.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   # to test support for sdk 1, uncomment this line
   # spec.add_development_dependency 'aws-sdk', '~> 1'
   # to test sdk 2 use this one
+  spec.add_dependency 'msgpack'
```

Comment:
I would try to make this an optional dependency... We want to be really careful about adding any additional dependencies... I guess with JSON built-in and only 6% better not sure if it is worth it... If so definitely feels more like an opt-in feature vs the default.

Thoughts?

Reply:
Agreed but I want to see how msg_pack compares when we increase the batch size of save_report.

Reply:
yeah I think you could just not have it as a dependency and then have something like a flag that would use it, and in that case it would require the user to explicitly add msgpack to their gemfile...

All that being said for testing it is obviously totally fine.

Reply:
Makes sense.